### ISSUE TITLE
Sorgenfrey line (S43) is not LOTS

### ISSUE THREAD

**Issue by Jianing-Song** [pi-base/data#770]
Please forgive me if this has already been mentioned in previous issues.

MathSE has a link that mentions the proof that the Sorgenfrey line is not LOTS: https://math.stackexchange.com/questions/1603512/sorgenfrey-line-is-not-orderable. Unfortunately, this question has been closed. (I don't think this closedness is justified, though.) Perhaps the better thing to do is to add the property "has $G_\delta$ diagonal", add the fact that the Sorgenfrey line has $G_\delta$ diagonal, and then add the theorem LOTS+has $G_\delta$ diagonal$\Rightarrow$metrizable (and the Sorgenfrey line shows that the condition cannot be weakened to GO-space).

What do you think?

**Comment by Jianing-Song** [comment]
Some theorems about "has $G_\delta diagonal" should this property be added:

LOTS + has $G_\delta diagonal $\Rightarrow$ metrizable: mentioned above.

submetrizable $\Rightarrow$ has $G_\delta diagonal: a finer topology of a topology with $G_\delta diagonal also has $G_\delta diagonal. This would also show that [LOTS + submetrizable + ~metrizable](https://topology.pi-base.org/spaces?q=LOTS%2BSubmetrizable%2B%7EMetrizable) is impossible.

has $G_\delta diagonal + has multple points $\Rightarrow$ ~indiscrete: obvious. But this is far too weak. Can the property "has $G_\delta diagonal" imply some separation axioms?

**Comment by prabau** [comment]
Yeah, I have been wanting to add this using Lutzer's famous theorem that LOTS with a $G_\delta$ diagonal are metrizable.  That result in itself is not so easy; at least it makes use of various other notions not currently in pi-base.

But as you suggest, a first step would be to introduce the notion of "has a $G_\delta$ diagonal" and related theorems.  There is a lot to explore here. It would be great if you want to do it.

Dan Ma's topology blog has many posts about the topic:
https://dantopology.wordpress.com/tag/g-delta-diagonal/

Note: Lutzer's theorem relies on various other concepts.  But its review in zbmath (https://zbmath.org/0177.50703) presents a shorter/easier? proof.  I don't understand all that at this point, but could be useful when we get to that theorem.  Maybe there is another account of it somewhere else.

**Comment by Jianing-Song** [comment]
> Yeah, I have been wanting to add this using Lutzer's famous theorem that LOTS with a G δ diagonal are metrizable for a long time. That result in itself is not so easy; at least it makes use of various other notions not currently in pi-base.
> 
> But as you suggest, a first step would be to introduce the notion of "has a G δ diagonal" and related theorems. There is a lot to explore here. It would be great if you want to do it.
> 
> Dan Ma's topology blog has many posts about the topic: https://dantopology.wordpress.com/tag/g-delta-diagonal/
> 
> Note: Lutzer's theorem relies on various other concepts. But its review in zbmath (https://zbmath.org/0177.50703) presents a shorter/easier? proof. I don't understand all that at this point, but could be useful when we get to that theorem. Maybe there is another account of it somewhere else.
> 
> Just to be clear, I think it would be best to start without Lutzer's theorem and just work on G_delta diagonal at first.

Yes, I totally agree that it is better to work separately on "has a $G_\delta$ diagonal" first :)

**Comment by Almanzoris** [comment]
Yeah, I have also thought that "has a $G_{\delta}$ diagonal" could be added when making T534.

**Comment by felixpernegger** [comment]
All of the things mentioned in this issue are in pibase now.
